schema.sql

```diff
@@ -0,0 +1,14 @@
+DROP DATABASE IF EXISTS photo_gallery;
+CREATE DATABASE photo_gallery;
+
+CREATE TABLE listings (
+    listing_id SERIAL PRIMARY KEY
+);
+
+CREATE TABLE photos (
+    listing_id INTEGER NOT NULL,
+    photo_id SERIAL NOT NULL PRIMARY KEY, // BIG SERIAL?
```

Comment:
I'm not sure if the `//` style comments work in .sql documents. [PostgreSQL Docs on Comments](https://www.postgresql.org/docs/8.0/sql-syntax.html#SQL-SYNTAX-COMMENTS)